src/CommunityToolkit.Maui/Views/AvatarView.shared.cs

```diff
@@ -321,9 +355,47 @@ void HandleFontChanged()
 		InvalidateMeasureInternal(InvalidationTrigger.MeasureChanged);
 	}
 
+	static string DefaultGravatarName(DefaultGravatarImage defaultGravatar)
+			=> defaultGravatar switch
+			{
+				DefaultGravatarImage.FileNotFound => "404",
+				DefaultGravatarImage.MysteryPerson => "mp",
+				_ => $"{defaultGravatar}".ToLower(),
+			};
+
+	static string GetMd5Hash(string str)
+	{
+		using MD5 md5 = MD5.Create();
+		byte[] hash = md5.ComputeHash(Encoding.UTF8.GetBytes(str));
```

Comment:
Can we use `Span<byte>`? Not sure about the size of this array

Reply:
Changed whole method to use `Span<byte>` and `BitConverter`.